packages/config/config/devices/0x0113/lrm-as.json

```diff
@@ -17,15 +17,23 @@
 	},
 	"associations": {
 		"1": {
-			"label": "Group 1",
+			"label": "Controller",
```

Comment:
This should be called Lifeline. Please check the style guide in the documentation and ensure that these changes conform.